js/util/glossary/terms.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable */
```

Comment:
I was kind of thinking this might be easiest to maintain as a markdown file that inherits the `json.html` jekyll layout -- what do you think?

Reply:
Ooo yeah, that is smart.